common/evidence.py

```diff
@@ -225,18 +242,46 @@ def read_path(path: str, spark_instance) -> DataFrame:
 def import_trait_mappings() -> DataFrame:
     """Load the remote trait mappings file to a Spark dataframe."""
 
-    remote_trait_mappings_url = (
-        'https://raw.githubusercontent.com/opentargets/curation/22.09.1/mappings/disease/manual_string.tsv'
-    )
+    remote_trait_mappings_url = "https://raw.githubusercontent.com/opentargets/curation/22.09.1/mappings/disease/manual_string.tsv"
```

Comment:
I see that this PR only changes the style of this line, and not the contents, but I still have to ask: is there a reason this URL is hardcoded here, rather than being stored somewhere in a config?

Reply:
No, and i think i have already raised to @ireneisdoomed that this practice needs to be reviewed. This path should come from the config.

Reply:
@DSuveges created a ticket about this and it is scoped for the next release

Reply:
Thank you! Since there is a ticket, I'm marking this thread as resolved